### PR TITLE
(DOCS-3864) Updated PHP example to use up to date API

### PR DESCRIPTION
**Pull request by CameronHall**
### What does this PR do?
Updated PHP example to use up-to-date API

### Motivation
A colleague linked me to this doc and I was annoyed that it didn't work when I copied the example.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by jtappa** [comment]
Hi @CameronHall, I'm sorry to hear you bumped into an old example. I really appreciate you taking action to update it though! I've added a Jira card for us to track this internally, and we'll get an SME to confirm the change before merging.

File: content/en/tracing/guide/add_span_md_and_graph_it.md [hunk 1]
```diff
@@ -200,13 +200,11 @@ The Datadog UI uses tags to set span level metadata. Custom tags may be set for
...
-
   class ShoppingCartController extends Controller
   {
       public shoppingCartAction (Request $request) {
           // Get the currently active span
-          $span = GlobalTracer::get()->getActiveSpan();
+          $span = \DDTrace\active_span();
           if (null !== $span) {
               // customer_id -> 254889
               $span->setTag('customer_id', $request->get('customer_id'));
```
Comment (labbati) [inline content/en/tracing/guide/add_span_md_and_graph_it.md:210]:
```suggestion
              $span->meta['customer_id'] = $request->get('customer_id');
```

**Review by labbati — CHANGES_REQUESTED** [review]
Just a minor change, as the new api has a different way to set tags. Other than this it looks good to me! Thanks for the fix

**Comment by CameronHall** [comment]
I did that originally before I force pushed 🤦🏼‍♂️😂

**Review by labbati — APPROVED** [review]
Thank you @CameronHall

**Review by kayayarai — APPROVED** [review]

**Pull request merged**